.gitignore

```diff
@@ -6,3 +6,8 @@ _static
 _templates
 
 TODO
+*.pyc
```

Comment:
That's fair, I actually didn't know a global gitignore was possible :smile:. I will revert those changes